>From 97afaddb4f66a81dc1aac30a8342f6322d1b48b1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?David=20Pr=C3=A9vot?= <taffit@debian.org>
Date: Thu, 20 Dec 2012 10:39:32 -0400
Subject: [PATCH] Do not terminate SEE ALSO list with a period

Follow man-pages(7) conventions.
---
 scripts/checkbashisms.1     |    2 +-
 scripts/cowpoke.1           |    2 +-
 scripts/cvs-debc.1          |    5 ++---
 scripts/cvs-debi.1          |    5 ++---
 scripts/cvs-debrelease.1    |    5 ++---
 scripts/cvs-debuild.1       |    5 ++---
 scripts/dcmd.1              |    2 +-
 scripts/dcontrol.pl         |    2 +-
 scripts/debc.1              |    5 ++---
 scripts/debchange.1         |    5 ++---
 scripts/debclean.1          |    5 ++---
 scripts/debcommit.pl        |    2 +-
 scripts/debdiff.1           |    5 ++---
 scripts/debi.1              |    5 ++---
 scripts/debrelease.1        |    5 ++---
 scripts/debrsign.1          |    5 ++---
 scripts/debsign.1           |    5 ++---
 scripts/debuild.1           |    3 +--
 scripts/dget.pl             |    2 +-
 scripts/dpkg-depcheck.1     |    5 ++---
 scripts/dpkg-genbuilddeps.1 |    8 ++++----
 scripts/dscverify.1         |    5 ++---
 scripts/grep-excuses.1      |    2 +-
 scripts/nmudiff.1           |    5 ++---
 scripts/plotchangelog.1     |    2 +-
 scripts/rmadison.pl         |    2 +-
 scripts/uscan.1             |    5 ++---
 scripts/uupdate.1           |    6 +++---
 28 files changed, 49 insertions(+), 66 deletions(-)

diff --git a/scripts/checkbashisms.1 b/scripts/checkbashisms.1
index 03d403e..6df5f3c 100644
--- a/scripts/checkbashisms.1
+++ b/scripts/checkbashisms.1
@@ -61,7 +61,7 @@ unreadable or not found.  The warning message will give details.
 4
 No bashisms were detected in a bash script.
 .SH "SEE ALSO"
-.BR lintian (1).
+.BR lintian (1)
 .SH AUTHOR
 \fBcheckbashisms\fR was originally written as a shell script by Yann Dirson
 <\fIdirson@debian.org\fR> and rewritten in Perl with many more features by
diff --git a/scripts/cowpoke.1 b/scripts/cowpoke.1
index f665b70..2643425 100644
--- a/scripts/cowpoke.1
+++ b/scripts/cowpoke.1
@@ -264,7 +264,7 @@ of uncommitted work.
 .BR cowbuilder (1),
 .BR pbuilder (1),
 .BR ssh-agent (1),
-.BR sudoers (5).
+.BR sudoers (5)
 
 .SH AUTHOR
 .B cowpoke
diff --git a/scripts/cvs-debc.1 b/scripts/cvs-debc.1
index 5ed6029..8b3f49a 100644
--- a/scripts/cvs-debc.1
+++ b/scripts/cvs-debc.1
@@ -58,9 +58,8 @@ Show help message and version information respectively.
 .SH "SEE ALSO"
 .BR cvs-buildpackage (1),
 .BR cvs-debi (1),
-.BR cvs-debuild (1)
-and
-.BR debc (1).
+.BR cvs-debuild (1),
+.BR debc (1)
 .SH AUTHOR
 \fBcvs-buildpackage\fR was written by Manoj Srivastava, and the
 current version of \fBdebi\fR was written by Julian Gilbey
diff --git a/scripts/cvs-debi.1 b/scripts/cvs-debi.1
index af92edf..81b2bb1 100644
--- a/scripts/cvs-debi.1
+++ b/scripts/cvs-debi.1
@@ -62,9 +62,8 @@ Show help message and version information respectively.
 .SH "SEE ALSO"
 .BR cvs-buildpackage (1),
 .BR cvs-debc (1)
-.BR cvs-debuild (1)
-and
-.BR debi (1).
+.BR cvs-debuild (1),
+.BR debi (1)
 .SH AUTHOR
 \fBcvs-buildpackage\fR was written by Manoj Srivastava, and the
 current version of \fBdebi\fR was written by Julian Gilbey
diff --git a/scripts/cvs-debrelease.1 b/scripts/cvs-debrelease.1
index 2ce0b01..446eddc 100644
--- a/scripts/cvs-debrelease.1
+++ b/scripts/cvs-debrelease.1
@@ -63,9 +63,8 @@ first option given on the command-line.
 Show help message and version information respectively.
 .SH "SEE ALSO"
 .BR cvs-buildpackage (1),
-.BR debrelease (1)
-and
-.BR cvs-debuild (1).
+.BR debrelease (1),
+.BR cvs-debuild (1)
 .SH AUTHOR
 \fBcvs-buildpackage\fR was written by Manoj Srivastava, and the
 current version of \fBdebrelease\fR was written by Julian Gilbey
diff --git a/scripts/cvs-debuild.1 b/scripts/cvs-debuild.1
index 2999643..bd4712a 100644
--- a/scripts/cvs-debuild.1
+++ b/scripts/cvs-debuild.1
@@ -53,8 +53,7 @@ These should not be needed, but it is provided nevertheless.
 .SH "SEE ALSO"
 .BR cvs-buildpackage (1),
 .BR debuild (1),
-.BR dpkg-buildpackage (1)
-and
-.BR lintian (1).
+.BR dpkg-buildpackage (1),
+.BR lintian (1)
 .SH AUTHOR
 This program was written by Julian Gilbey <jdg@debian.org>.
diff --git a/scripts/dcmd.1 b/scripts/dcmd.1
index 6aa0b49..e3e7180 100644
--- a/scripts/dcmd.1
+++ b/scripts/dcmd.1
@@ -97,7 +97,7 @@ $
 
 .SH "SEE ALSO"
 .BR dpkg-source (1),
-.BR dpkg-genchanges (1).
+.BR dpkg-genchanges (1)
 .SH AUTHOR
 This program was written by Romain Francoise <rfrancoise@debian.org> and
 is released under the GPL, version 2 or later.
diff --git a/scripts/dcontrol.pl b/scripts/dcontrol.pl
index deac5c6..0e6bee3 100755
--- a/scripts/dcontrol.pl
+++ b/scripts/dcontrol.pl
@@ -288,4 +288,4 @@ of the License, or (at your option) any later version.
 
 =head1 SEE ALSO
 
-B<apt-cache>(1).
+B<apt-cache>(1)
diff --git a/scripts/debc.1 b/scripts/debc.1
index 2a54761..cd320e8 100644
--- a/scripts/debc.1
+++ b/scripts/debc.1
@@ -114,9 +114,8 @@ which check their value, as described in their respective manpages and
 in \fBdevscripts.conf\fR(5).
 .SH "SEE ALSO"
 .BR dpkg-deb (1),
-.BR debdiff (1)
-and
-.BR devscripts.conf (5).
+.BR debdiff (1),
+.BR devscripts.conf (5)
 .SH AUTHOR
 Julian Gilbey <jdg@debian.org>, based on an original script by
 Christoph Lameter <clameter@debian.org>.
diff --git a/scripts/debchange.1 b/scripts/debchange.1
index b989780..b3b9a8b 100644
--- a/scripts/debchange.1
+++ b/scripts/debchange.1
@@ -470,9 +470,8 @@ by \fBsensible-editor\fR.
 .BR debclean (1),
 .BR dupload (1),
 .BR dput (1),
-.BR debc (1)
-and
-.BR devscripts.conf (5).
+.BR debc (1),
+.BR devscripts.conf (5)
 .SH AUTHOR
 The original author was Christoph Lameter <clameter@debian.org>.
 Many substantial changes and improvements were made by Julian Gilbey
diff --git a/scripts/debclean.1 b/scripts/debclean.1
index c43bf32..ed45ac9 100644
--- a/scripts/debclean.1
+++ b/scripts/debclean.1
@@ -105,9 +105,8 @@ variables, and will therefore affect all \fBdevscripts\fR scripts
 which check their value, as described in their respective manpages and
 in \fBdevscripts.conf\fR(5).
 .SH "SEE ALSO"
-.BR debuild (1)
-and
-.BR devscripts.conf (5).
+.BR debuild (1),
+.BR devscripts.conf (5)
 .SH AUTHOR
 Christoph Lameter <clameter@debian.org>;
 modifications by Julian Gilbey <jdg@debian.org>.
diff --git a/scripts/debcommit.pl b/scripts/debcommit.pl
index f8a08e6..20a4179 100755
--- a/scripts/debcommit.pl
+++ b/scripts/debcommit.pl
@@ -862,6 +862,6 @@ Joey Hess <joeyh@debian.org>
 
 =head1 SEE ALSO
 
-svnpath(1), debchange(1).
+B<svnpath>(1), B<debchange>(1).
 
 =cut
diff --git a/scripts/debdiff.1 b/scripts/debdiff.1
index c0cb337..181ec16 100644
--- a/scripts/debdiff.1
+++ b/scripts/debdiff.1
@@ -216,9 +216,8 @@ be 255.
 .BR dpkg-deb (1),
 .BR wdiff (1),
 .BR interdiff (1),
-.BR diffstat (1)
-and
-.BR devscripts.conf (5).
+.BR diffstat (1),
+.BR devscripts.conf (5)
 .SH AUTHOR
 \fBdebdiff\fR was originally written as a shell script by Yann Dirson
 <dirson@debian.org> and rewritten in Perl with many more features by
diff --git a/scripts/debi.1 b/scripts/debi.1
index 2fb3bdf..779462a 100644
--- a/scripts/debi.1
+++ b/scripts/debi.1
@@ -130,9 +130,8 @@ variables, and will therefore affect all \fBdevscripts\fR scripts
 which check their value, as described in their respective manpages and
 in \fBdevscripts.conf\fR(5).
 .SH "SEE ALSO"
-.BR debpkg (1)
-and
-.BR devscripts.conf (5).
+.BR debpkg (1),
+.BR devscripts.conf (5)
 .SH AUTHOR
 \fBdebi\fR was originally written by Christoph Lameter
 <clameter@debian.org>.  The now-defunct script \fBdebit\fR was
diff --git a/scripts/debrelease.1 b/scripts/debrelease.1
index 1758f48..c3c04b8 100644
--- a/scripts/debrelease.1
+++ b/scripts/debrelease.1
@@ -131,9 +131,8 @@ which check their value, as described in their respective manpages and
 in \fBdevscripts.conf\fR(5).
 .SH "SEE ALSO"
 .BR dupload (1),
-.BR dput (1)
-and
-.BR devscripts.conf (5).
+.BR dput (1),
+.BR devscripts.conf (5)
 .SH AUTHOR
 Julian Gilbey <jdg@debian.org>, based on the original \fBrelease\fR
 script by Christoph Lameter <clameter@debian.org>.
diff --git a/scripts/debrsign.1 b/scripts/debrsign.1
index 1a17131..5f6ccf9 100644
--- a/scripts/debrsign.1
+++ b/scripts/debrsign.1
@@ -65,9 +65,8 @@ purpose.  The currently recognised variables are:
 Equivalent to passing \fB\-\-path\fR on the command line (see above.)
 .SH "SEE ALSO"
 .BR debsign (1),
-.BR dpkg-architecture (1)
-and
-.BR ssh (1).
+.BR dpkg-architecture (1),
+.BR ssh (1)
 .SH AUTHOR
 This program was written by Julian Gilbey <jdg@debian.org> and is
 copyright under the GPL, version 2 or later.
diff --git a/scripts/debsign.1 b/scripts/debsign.1
index 36575c7..3d0c616 100644
--- a/scripts/debsign.1
+++ b/scripts/debsign.1
@@ -140,9 +140,8 @@ the option.
 .BR sha1sum (1),
 .BR sha256sum (1),
 .BR gpg (1),
-.BR scp (1)
-and
-.BR devscripts.conf (5).
+.BR scp (1),
+.BR devscripts.conf (5)
 .SH AUTHOR
 This program was written by Julian Gilbey <jdg@debian.org> and is
 copyright under the GPL, version 2 or later.
diff --git a/scripts/debuild.1 b/scripts/debuild.1
index 2cd8a41..07a5072 100644
--- a/scripts/debuild.1
+++ b/scripts/debuild.1
@@ -417,8 +417,7 @@ sponsored upload.
 .BR dpkg-statoverride (8),
 .BR su (1),
 .BR sudo (1),
-.BR super (1)
-and
+.BR super (1),
 .BR devscripts.conf (5).
 .SH AUTHOR
 The original \fBdebuild\fR program was written by Christoph Lameter
diff --git a/scripts/dget.pl b/scripts/dget.pl
index af4d034..d0d93aa 100755
--- a/scripts/dget.pl
+++ b/scripts/dget.pl
@@ -653,4 +653,4 @@ of the License, or (at your option) any later version.
 =head1 SEE ALSO
 
 B<apt-get>(1), B<debcheckout>(1), B<debdiff>(1), B<dpkg-source>(1), B<curl>(1),
-B<wget>(1).
+B<wget>(1)
diff --git a/scripts/dpkg-depcheck.1 b/scripts/dpkg-depcheck.1
index 5cae524..2c17916 100644
--- a/scripts/dpkg-depcheck.1
+++ b/scripts/dpkg-depcheck.1
@@ -118,9 +118,8 @@ quoting, because of the vagaries of shell quoting!
 .SH "SEE ALSO"
 .BR dpkg (1),
 .BR strace (1),
-.BR update-alternatives (8)
-and
-.BR devscripts.conf (5).
+.BR update-alternatives (8),
+.BR devscripts.conf (5)
 .SH "COPYING"
 Copyright 2001 Bill Allombert <ballombe@debian.org>.
 Modifications copyright 2002,2003 Julian Gilbey <jdg@debian.org>.
diff --git a/scripts/dpkg-genbuilddeps.1 b/scripts/dpkg-genbuilddeps.1
index 2f2f321..7a7f5cd 100644
--- a/scripts/dpkg-genbuilddeps.1
+++ b/scripts/dpkg-genbuilddeps.1
@@ -28,11 +28,11 @@ All this program itself does is essentially to run the command:
   dpkg-depcheck \-b dpkg-buildpackage \-us \-uc \-b \-rfakeroot [arg ...]
 .fi
 .SH "SEE ALSO"
+.BR dpkg-depcheck (1),
+.BR fakeroot (1)
+
 .B The Debian Policy Manual,
-sections on Build-Depends etc.,
-.BR dpkg-depcheck (1)
-and
-.BR fakeroot (1).
+sections on Build-Depends etc.
 .SH AUTHOR
 The original \fBdpkg-genbuilddeps\fR was written by Ben Collins
 <bcollins@debian.org>.  The current version is a simple wrapper around
diff --git a/scripts/dscverify.1 b/scripts/dscverify.1
index 06803cf..3d5e8ce 100644
--- a/scripts/dscverify.1
+++ b/scripts/dscverify.1
@@ -71,9 +71,8 @@ locations:
 
 - /usr/share/keyrings/debian-maintainers.gpg
 .SH "SEE ALSO"
-.BR gpg (1)
-and
-.BR devscripts.conf (5).
+.BR gpg (1),
+.BR devscripts.conf (5)
 
 .SH AUTHOR
 \fBdscverify\fR was written by Roderick Schertler <roderick@argon.org>
diff --git a/scripts/grep-excuses.1 b/scripts/grep-excuses.1
index a03ae87..eeb5c58 100644
--- a/scripts/grep-excuses.1
+++ b/scripts/grep-excuses.1
@@ -36,7 +36,7 @@ purpose.  The currently recognised variable is:
 The default maintainer, email or package to grep for if none is
 specified on the command line.
 .SH "SEE ALSO"
-.BR devscripts.conf (5).
+.BR devscripts.conf (5)
 .SH AUTHOR
 Joey Hess <joeyh@debian.org>;
 modifications by Julian Gilbey <jdg@debian.org>.
diff --git a/scripts/nmudiff.1 b/scripts/nmudiff.1
index c1f5001..9e72963 100644
--- a/scripts/nmudiff.1
+++ b/scripts/nmudiff.1
@@ -107,9 +107,8 @@ If this is set, specifies a \fBsendmail\fR command to use instead of
 \fI/usr/sbin/sendmail\fR.  Same as the \fB\-\-sendmail\fR command line option.
 .SH "SEE ALSO"
 .BR debdiff (1),
-.BR sensible-editor (1)
-and
-.BR devscripts.conf (5).
+.BR sensible-editor (1),
+.BR devscripts.conf (5)
 .SH AUTHOR
 \fBnmudiff\fR was written and is copyright 2006 by Steinar
 H. Gunderson and modified by Julian Gilbey <jdg@debian.org>.  The
diff --git a/scripts/plotchangelog.1 b/scripts/plotchangelog.1
index 744e70b..ada427d 100644
--- a/scripts/plotchangelog.1
+++ b/scripts/plotchangelog.1
@@ -122,6 +122,6 @@ These are
 commands which will be prepended to any such commands given on the
 command line.
 .SH "SEE ALSO"
-.BR devscripts.conf (5).
+.BR devscripts.conf (5)
 .SH AUTHOR
 Joey Hess <joey@kitenet.net>
diff --git a/scripts/rmadison.pl b/scripts/rmadison.pl
index 76a622f..73521a7 100755
--- a/scripts/rmadison.pl
+++ b/scripts/rmadison.pl
@@ -331,7 +331,7 @@ enable plain-text output.
 
 =head1 SEE ALSO
 
-madison-lite(1), dak(1).
+B<madison-lite>(1), B<dak>(1)
 
 =head1 AUTHOR
 
diff --git a/scripts/uscan.1 b/scripts/uscan.1
index 1aa2f38..0a6191c 100644
--- a/scripts/uscan.1
+++ b/scripts/uscan.1
@@ -547,9 +547,8 @@ avoid this behaviour, change the non-version-number groups to be
 .SH "SEE ALSO"
 .BR dpkg (1),
 .BR perlre (1),
-.BR uupdate (1)
-and
-.BR devscripts.conf (5).
+.BR uupdate (1),
+.BR devscripts.conf (5)
 .SH AUTHOR
 The original version of \fBuscan\fR was written by Christoph Lameter
 <clameter@debian.org>.  Significant improvements, changes and bugfixes
diff --git a/scripts/uupdate.1 b/scripts/uupdate.1
index 1aa7bc2..41c86ca 100644
--- a/scripts/uupdate.1
+++ b/scripts/uupdate.1
@@ -171,9 +171,9 @@ the Debian revision to \*(lq0ubuntu1\*(rq.  You may change
 .BR debuild (1),
 .BR fakeroot (1),
 .BR patch (1),
-.I The Debian Policy Manual
-and
-.BR devscripts.conf (5).
+.BR devscripts.conf (5)
+
+.B The Debian Policy Manual
 .SH AUTHOR
 The original version of \fBuupdate\fR was written by Christoph Lameter
 <clameter@debian.org>.  Several changes and improvements have been
-- 
1.7.10.4

